src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathfilter/cyclomaticcomplexitycheck/InputSuppressionXpathFilterCyclomaticTwo.java

```diff
@@ -0,0 +1,25 @@
+package com.puppycrawl.tools.checkstyle.filters.suppressionxpathfilter.cyclomaticcomplexitycheck;
+
+public class InputSuppressionXpathFilterCyclomaticTwo {
+
+
+    public void foo2() {
```

Comment:
as we expect only one violation in file, please put trailing comment where violation is expected.
"// violation".
Please do this for all input files.

Reply:
I used `//warn`, because [AbstractModuleTestSupport](https://github.com/checkstyle/checkstyle/blob/master/src/it/java/com/google/checkstyle/test/base/AbstractModuleTestSupport.java#L78) requires this comment